docs/community.aws.route53_module.rst

```diff
@@ -26,7 +26,7 @@ Requirements
 The below requirements are needed on the host that executes this module.
 
 - python >= 2.6
-- boto3
+- boto
```

Comment:
This needs to be boto3, apparently I missed that he'd updated the RST and not the doc fragment.

Reply:
added in #417 but I have to sign off for the night and ci isnt done there yet